devices/Scd30/Entities/Measurement.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) 2017 The nanoFramework project contributors
+// See LICENSE file in the project root for full license information.
+
+using System;
+using System.Buffers.Binary;
+using UnitsNet;
+
+namespace Iot.Device.Scd30.Entities
+{
+    /// <summary>
+    /// Provides the measurement values read from the sensor.
+    /// </summary>
+    public class Measurement
+    {
+        internal Measurement(SpanByte data)
+        {
+            var bytes = data.ToArray();
+            Co2ConcentrationInPpm = BinaryPrimitives.ReadSingleBigEndian(bytes);
+            Temperature = Temperature.FromDegreesCelsius(BinaryPrimitives.ReadSingleBigEndian(new SpanByte(bytes, 4, 4)));
+            RelativeHumidity = RelativeHumidity.FromPercent(BinaryPrimitives.ReadSingleBigEndian(new SpanByte(bytes, 8, 4)));
+        }
+
+        /// <summary>
+        /// Gets CO2 concentration (0-10000 ppm).
+        /// </summary>
+        /// <remarks>
+        /// We are using the <see cref="float"/> unit since UnitsNet.Turbidity is missing an implementation for PPM.
+        /// </remarks>
+        public float Co2ConcentrationInPpm { get; private set; }
```

Comment:
Perfect, thanks. I'll use this unit. I found Turbidity where they previously discussed in a pull request to add PPM but left it out because it has no foreseen usage.